otx/algorithms/segmentation/adapters/mmseg/__init__.py

```diff
@@ -33,6 +32,7 @@
     StepScalarScheduler,
     SupConDetConB,
 )
+from .nncf import CustomstepLrUpdaterHook
```

Comment:
Why would we need to import nncf here?

Reply:
Without this import, `NNCFTask` cannot find `CustomstepLRUpdaterHook` in registry.

Reply:
Is the NNCFTask you are referring to a common Task?

Reply:
Hmm no, it's for `SegmentationNNCFTask`, sorry for my lazy to type `NNCFTask`